### ISSUE TITLE
Move Line Up / Down Error

### ISSUE THREAD

**Issue by genericandy** [adobe/brackets#11564]
When I try to use the keyboard shortcut for Move Lines Up / Down (Ctrl+Shift+Up/Down) in a Javascript file, it selects the next line as well with each key click and moves that line too. This appears to be connected to the keyboard shortcut & not the command itself. If I use the command on the pulldown menu, it works as expected.

I tried with Extensions disabled & it still occurs.

I remapped 'edit.lineUp' and 'edit.lineDown' to 'ctrl+alt+up/down' using the remap keys option. After remapping, those shortcuts worked properly without the selection bug. So the issue seems to be specific to ctrl+shift+up/down.

Details:
Windows 10
Brackets 1.4.0 - latest build
Only JavaScript files. (SCSS, HTML, JSON, PHP all work properly)

I am not sure if this is relevant or not, but I do not remember any issues like this prior to upgrading to Win 10.

**Comment by redmunds** [comment]
Do you have any extensions installed? Maybe that shortcut is getting mapped to another command. Does it still happen after *Debug > Reload Without Extensions* ?

Try installing the Display Shortcuts extension to see shortcut mappings.

**Comment by genericandy** [comment]
Yeah I did that. I tried a reload without Extensions & it still occurs.

**Comment by genericandy** [comment]
I have worked out what is causing this in Windows 10 but I do not have a solution to fix it as it seems to be an internal error for brackets itself.

If the setting "showCursorWhenSelecting" is set to true, it causes the issue I am seeing. With it set to false, the issue goes away.
